### ISSUE TITLE
ln: /usr/local/bin/dots: Permission denied 

### ISSUE THREAD

**Issue by mrcarl79** [MatthewMueller/dots#18]
Have I missed anything here? fresh yosi install, ran command, entered admin password... Permission denied?

$ (mkdir -p /tmp/dots && cd /tmp/dots && curl -L https://github.com/matthewmueller/dots/archive/master.tar.gz | tar zx --strip 1 && sh ./install.sh)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   127    0   127    0     0     92      0 --:--:--  0:00:01 --:--:--    92
100 12806  100 12806    0     0   4160      0  0:00:03  0:00:03 --:--:-- 24071
Password:
ln: /usr/local/bin/dots: Permission denied

**Comment by matthewmueller** [comment]
hmm. can you try running:

```
 (mkdir -p /tmp/dots && cd /tmp/dots && curl -L https://github.com/matthewmueller/dots/archive/master.tar.gz | tar zx --strip 1 && sudo sh ./install.sh)
```

?

**Comment by mrcarl79** [comment]
Yeah that got it going, when I started it by running dots boot osx it went though a number of steps but then hit an error and halted. FYI I'm running a brand new macbook retina with OS X 10.10.2, in the end I will probably will mod these scripts to suit my self. But on a fresh machine that I'm not fully using yet I was interested to see what your script did out of the box.

Thanks, Carl

Enabling snap-to-grid for icons on the desktop and in other icon views
Set: Entry, ":FK_StandardViewSettings:IconViewSettings:arrangeBy", Does Not Exist
